apps/binding.go

```diff
@@ -1,39 +1,128 @@
 package apps
 
+// Binding is the main way for an App to attach its functionality to the
+// Mattermost UI.
+//
+// An App returns the bindings in response to the "bindings" call, that it must
+// implement, and can customize in its Manifest. For each context in which it is
```

Comment:
How can we enforce "that it must implement" - maybe add a section in the doc that calls out the "must haves" wrt to functionality?

Reply:
https://github.com/mattermost/mattermost-plugin-apps/blob/d780ff7b40767bb504a3a72ccaf2c40d87189c30/apps/manifest.go#L23